packages/web/webpack/build.ts

```diff
@@ -0,0 +1,31 @@
+import webpack from 'webpack';
+import yargs from 'yargs/yargs';
+import { hideBin } from 'yargs/helpers';
+import { clientConfig } from './client';
+import { serverConfig } from './server';
+
+const args = yargs(hideBin(process.argv))
```

Comment:
maybe `cli/build.ts` ?

Reply:
This is the only cli script in the web package, so I wouldn't make a separate `cli` folder for it, at least for now